ConanToolWindowControl.xaml

```diff
@@ -36,7 +36,13 @@
         <CheckBox x:Name="ShowPackagesCheckbox" Click="ShowPackagesCheckbox_Click" Content="List only installed libraries" Grid.Row="1" Grid.ColumnSpan="2" Margin="5" />
 
         <ListView x:Name="PackagesListView" SelectionChanged="ListView_SelectionChanged" Grid.Row="2" Grid.ColumnSpan="2" Margin="5" ></ListView>
-
+        <StackPanel x:Name="UnsupportedProjectType" Visibility="Collapsed" Grid.Row="3" Grid.ColumnSpan="2">
+            <TextBlock Visibility="Visible" FontSize="18" FontWeight="Bold">Unsupported Project Type</TextBlock>
+            <TextBlock>

```

Comment:
```suggestion
        <StackPanel x:Name="UnsupportedProjectType" Visibility="Collapsed" Grid.Row="3" Grid.ColumnSpan="2" Margin="5">
            <TextBlock Margin="5,0,5,5" Visibility="Visible" FontSize="18" FontWeight="Bold">Unsupported Project Type</TextBlock>
            <TextBlock Margin="5,0,5,5" TextWrapping="Wrap">

```

Reply:
I'm suggesting some style changes to add margins and also the text was overflowing in my case
Now it would look like:

![image](https://github.com/conan-io/conan-vs-extension/assets/5045666/39fa20f2-2de2-4ecd-bcbb-6aeebc984a84)